bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/ColumnViewerSelectionColorListener.java

```diff
@@ -0,0 +1,65 @@
+package org.eclipse.jface.viewers;
```

Comment:
Copyright Header is missing

Reply:
Mh. I still don't see the copyright header.

Reply:
Done